pkg/controller/instance/instance_controller.go

```diff
@@ -255,7 +256,25 @@ func (r *Reconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 		r.Recorder.Event(instance, "Normal", "PlanFinished", fmt.Sprintf("Execution of plan %s finished with status %s", newStatus.Name, newStatus.Status))
 	}
 
-	return reconcile.Result{}, nil
+	return computeTheReconcileResult(instance, time.Now), nil
+}
+
+// computeTheReconcileResult decides whether retry reconciliation or not
+// if plan was finished, reconciliation is not retried
+// for others it uses LastUpdatedTimestamp of a current plan
+// for plan updated less than a minute ago, the backoff would be a second, 10 minutes would be 10 seconds
```

Comment:
this seems challenging to read
> for plan updated less than a minute ago, the backoff would be a second, 10 minutes would be 10 seconds maximum backoff is one minute

I think what is meant is the initial backoff is 1 sec and that 1 min is the max backoff.   I can't make sense of 10 minutes would be 10 seconds

Reply:
I tried to clarify, let me know if it's better